src/etc/inc/auth.inc

```diff
@@ -1959,7 +1964,7 @@ function session_auth() {
 
 	/* Validate incoming login request */
 	$attributes = array('nas_identifier' => 'webConfigurator-' . gethostname());
-	if (isset($_POST['login']) && !empty($_POST['usernamefld']) && !empty($_POST['passwordfld'])) {
+	if (isset($_POST['login']) && !empty($_POST['usernamefld'])) {
```

Comment:
Why change this here? It would alter existing behavior for non-LDAP logins unrelated to this change.

Reply:
Hi,
The idea is remove password checks from here and let *_backed() functions handle password controls. 

I understand that this is a change in a critical part of the code however....Please let me know if you would like me to revert.